src/nginxconfig/util/deep_merge.js

```diff
@@ -24,22 +24,15 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-import common from '../../common';
-
-export default {
-    phpServer: `${common.php} 服務`,
-    phpMustBeEnabledOnOneSite: `必須在至少一個網站上啟用${common.php}才能配寘全域${common.php}設定。`,
-    phpBackupServer: `${common.php}備份服務器`,
-    tcp: 'TCP',
-    hhvmSocket: 'HHVM socket',
-    php5Socket: '5.x socket',
-    php70Socket: '7.0 socket',
-    php71Socket: '7.1 socket',
-    php72Socket: '7.2 socket',
-    php73Socket: '7.3 socket',
-    php74Socket: '7.4 socket',
-    php80Socket: '8.0 socket',
-    phpSocket: 'PHP socket',
-    custom: '自定义',
-    disabled: '禁用',
+export default (target, source) => {
+    const merge = (target, source) => {
+        Object.keys(source).forEach(function (key) {
```

Comment:
```suggestion
        Object.keys(source).forEach((key) => {
```
We can also use an arrow function here ;)

Reply:
Oh, no, I'm a complete disaster. :neckbeard: